src/core_plugins/timelion/public/webpackShims/jquery-flot.js

```diff
@@ -1,7 +1,16 @@
+const $ = require('jquery');
+
+if (window) window.jQuery = $;
```

Comment:
In your description, you say:
> instead of trying to enhance whatever version of jQuery it finds on window.

Doesn't the above line overwrite whatever version of `jQuery` was already defined on `window`? You probably have a good reason to do this, but it feels wrong to me.

Apparently we're doing the same think in the watcher UI...